src/posts/2021-05-21-google-io-learn-something-new.md

```diff
@@ -0,0 +1,82 @@
+
+<!-- Section: Workshops & Codelabs -->
+
+### Google Workshops & Codelabs
+
+Het is tijdens het evenement ook mogelijk om deel te nemen aan een of meerdere workshops. Iedere workshop wordt verzorgt door een Googler en neemt ongeveer een uur in beslag. Letterlijk voor ieder product of onderwerp is er wel een workshop beschikbaar. Kijk maar eens op de website onder het kopje [Learning Lab](https://events.google.com/io/learning-lab/?lng=en).
+
+![Google I/O Workshops](/assets/io-codelabs/io21-workshops.png)
+
+Meestal gaat de workshop gepaard met een verwante **Google Codelab**. Een Google Codelab is niks anders dan een tuturial of instructie waarmee je zelf, stap-voor-stap, een kleine applicatie of feature kan bouwen. Google stelt diverse codelabs beschikbaar op [codelabs.developers.google.com](https://codelabs.developers.google.com/) (gratis).
+
+### Tijd om iets (nieuws) te leren!
+
+De beste manier om technische skills te ontwikkelen, is door gewoon te doen. En een leuke manier om nieuwe trends te ontdekken, is door gewoon een workshop buiten je directe expertise te pakken (bijvoorbeeld Machine Learning of Flutter).
+
+Hier zijn enkele suggesties op een rij:
+
+<!-- prettier-ignore-start -->
+| Workshop | Onderwerp | &nbsp; |
+| - | - | - | - |
+{% 
+    for item in io21_workshops 
+%} | [{{ item.title }}]({{ item.url }}) | {{ item.topic }} | [video]({{ item.video_url }})&nbsp;&middot;&nbsp;[codelab]({{ item.codelab_url }}) |
+{% 
+    endfor 
+%}
+<!-- prettier-ignore-end -->
+
+**Bonus**: maak eerst even een [Google Developer Profile](https://developers.google.com/profile) aan en ontvang na het afronden van je eerste codelab, een badge.
+
+Dit zijn een aantal interessante ontwikkelingen die je in een workshop voorbij kan zien komen:
+
+- **Flutter** ken je waarschijnlijk al van cross-platform ontwikkeling voor Android en iOS. Wist je ook dat je nu met Flutter vanuit dezelfde codebase, ook kan ontwikkelen voor Desktop (Windows/Mac/Linux/ChromeOS) en Web?
+
+- **Vertex AI**: een nieuw (managed) _machine learning_ platform van Google Cloud. In de workshop wordt ingegaan op de tooling, waarmee je zelf model kan trainen en uitrollen met Vertex AI.
+
+- **App Actions & Google Assistant**: open een _feature_ van je Android app, direct via Google Assistant. Neem als voorbeeld de app BookMyDesk (om handige app om werkplek te reserveren op kantoor). Zou het dan niet makkelijk zijn, als je gewoon tegen Google Assistant zou kunnen zeggen "Hey Google, reserveer een werkplek voor morgen." ?
+
+- **Project Fugu**: dit is projectnaam voor een verzameling moderne (en toekomstige) web browser API's. Wist je dat je vanaf je mobiele browser, de _native_ deelfunctie van je mobiel kan aanroepen?
+
+Hopelijk ben je hiermee geinspireerd geraakt zelf ook een workshop of codelab te volgen. **Welke wordt het?**
+
+<!-- Section: Externe links en referenties -->
+
+### Externe links
+
+- [Bekijk de Google I/O Keynote](https://www.youtube.com/watch?v=L7c4wS7T_T8)
```

Comment:
- Other URLs are just what it is (not with; `Bekijk de ...`). And it's the recap, not the entire keynote itself; `Google I/O Keynote - recap` or `Google I/O in under 9 minutes`.
- Perhaps instead of the recap, have the full keynote & developer keynote linked?

Reply:
Updated